printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD

```diff
@@ -0,0 +1,53 @@
+![Design Image](./images/NotaSexboltimage.png)
+
+[Video of probing action](https://youtu.be/bdxmqTDDsMM)
+
+**BOM**
+
+ 1 x HeartK Voron 2.4 Endstop switch      
+ 4 x M2x8 Self Tapping screws      
+ 3 x 5x7x8 Sleeve Bearing      
+ 1 x M3 x 20mm round amuminum standoff post ( 5MM od)   
+      [Example of what to look for](https://www.aliexpress.com/item/2251832695751106.html?spm=a2g0o.productlist.0.0.39d572d1Ikaegt&algo_pvid=0ef968d6-c41a-4bf9-8728-aab261ae0f26&aem_p4p_detail=202207041141012168556211044480003127304&algo_exp_id=0ef968d6-c41a-4bf9-8728-aab261ae0f26-4&pdp_ext_f=%7B%22sku_id%22%3A%2265705928134%22%7D&pdp_npi=2%40dis%21USD%21%212.23%21%21%21%21%21%402132f35216569600607318207ec8b9%2165705928134%21sea). I did not buy from this store.. Just using them as a reference for the type of post you need.    
```

Comment:
please delete the `?` and everything after it in the link, it's all tracking info.

Reply:
so just `https://www.aliexpress.com/item/2251832695751106.html`

Reply:
done